.github/workflows/ci-additional.yaml

```diff
@@ -6,6 +6,9 @@ on:
   pull_request:
     branches:
       - "main"
+    paths:
+      - 'ci/**'
```

Comment:
Possibly we could move `ci` into `.github`; consolidate some of the top-level paths